src/routes/auth.js

```
@@ -35,7 +35,6 @@ router.patch(
   asyncWrapper(authController.updatePassword)
 )
 
-router.get('/confirm-email', asyncWrapper(authController.confirmEmail));
-
+router.get('/confirm-email/:confirmToken', asyncWrapper(authController.confirmEmail))
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)